examples/docs_snippets/docs_snippets/guides/components/index/1-help.txt

```diff
@@ -5,14 +5,16 @@ Usage: dg [OPTIONS] COMMAND [ARGS]...
  CLI for managing Dagster projects.                                                                                     
                                                                                                                         
 ╭─ Options ────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
-│ --clear-cache                           Clear the cache.                                                             │
-│ --rebuild-component-registry            Recompute and cache the set of available component types for the current     │
-│                                         environment. Note that this also happens automatically whenever the cache is │
-│                                         detected to be stale.                                                        │
-│ --install-completion                    Automatically detect your shell and install a completion script for the `dg` │
-│                                         command. This will append to your shell startup file.                        │
-│ --version                     -v        Show the version and exit.                                                   │
-│ --help                        -h        Show this message and exit.                                                  │
+│ --root-path                           PATH  Specify a directory to use to load the context for this command. This    │
```

Comment:
`root_path` is a little misleading because what we call the `root_path` internally is determined by searching up the directory hierarchy from the input path (currently CWD). I would just call this `--path`.